### PR TITLE
where does the director live page radio button

### PR DESCRIPTION
**Pull request by olaych**
Added where the director live radio buttons 
Added skeleton redirect pages depending on selected radio button

### CODE REVIEW

File: views/director-residential-address.html [hunk 1]
```diff
@@ -18,9 +18,51 @@
       
       <form action="" method="post">
 
+        <span class="govuk-caption-xl govuk-!-padding-bottom-1">Doe, John Doe</span>
+
```
Comment (jbestwickch) [inline views/director-residential-address.html:22]: Missing director name here as compared to prototype

File: views/director-residential-address.html [hunk 2]
```diff
@@ -18,9 +18,51 @@
       
       <form action="" method="post">
 
+        <span class="govuk-caption-xl govuk-!-padding-bottom-1">Doe, John Doe</span>
```
Comment (jbestwickch) [inline views/director-residential-address.html:21]: Director name is hard coded, should be passed in from filing I believe?

Comment (olaych) [reply views/director-residential-address.html:21]: Yes all the values are hardcoded as it is dependent on the previous page (which is yet to be done). So figured all the values will be populated once that is ready

Comment (jbestwickkainos) [reply views/director-residential-address.html:21]: Director name page is already implemented so we can pull name from filing. See occupation page html/controller for an example

Comment (olaych) [reply views/director-residential-address.html]: Name added

**Review by clewis1 — APPROVED** [review]
Looks good.

**Review by jbestwickch — APPROVED** [review]

**Pull request merged**